pandas/tests/io/test_sql.py

```diff
@@ -2148,6 +2148,10 @@ def test_to_sql_replace(self):
     def test_to_sql_append(self):
         self._to_sql_append()
 
+    def test_to_sql_method_multi(self):
+        # GH 29921
+        self._to_sql(method="multi")
```

Comment:
The machinery that we use to test SQL io makes writing this test very straightforward!

Let me know if you need something more indepth for this feature.